src/highlight/__tests__/HighlightCanvas-test.tsx

```diff
@@ -56,5 +55,23 @@ describe('HighlightCanvas', () => {
             const wrapper = getWrapper();
             expect(wrapper.find('canvas').hasClass('ba-HighlightCanvas')).toBe(true);
         });
+
+        test('should render, excluding 0 width and height values', () => {
+            const wrapper = getWrapper({ shapes: [mockRect, noWidthRect, noHeightRect] });
+            const instance = wrapper.instance() as HighlightCanvas;
+
+            instance.canvasRef = {
+                current: {
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+                    // @ts-ignore
```

Comment:
Why are these needed?

Reply:
Ah, this is because typescript will complain about the return value of `getContext`; the code still compiles and the test still runs, but I figured we could disable this typing error just in case